packages/manager/apps/key-management-service/src/components/credential/create/identities/tile/IdentitiesGroupTile.component.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { Dispatch, SetStateAction, useEffect, useState } from 'react';
+import { OsdsText } from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import { IdentityGroup } from '@/types/identity.type';
+import IdentitiesBaseTile from './IdentitiesBaseTile.component';
+import IdentitiesTileText from './IdentitiesTileText.component';
+
+type IdentitiesGroupTileProps = {
+  group: IdentityGroup;
+  selectedGroupList: IdentityGroup[];
+  setSelectedGroupList: Dispatch<SetStateAction<IdentityGroup[]>>;
+};
+const IdentitiesGroupTile = ({
+  group,
+  selectedGroupList,
+  setSelectedGroupList,
+}: IdentitiesGroupTileProps) => {
+  const { t } = useTranslation('key-management-service/credential');
+  const [isChecked, setIsChecked] = useState<boolean>(false);
+
+  useEffect(() => {
+    if (
+      selectedGroupList.find((groupInList) => groupInList.urn === group.urn)
+    ) {
+      setIsChecked(true);
```

Comment:
I think you may need to update the isChecked and uncheck it if the condition return false too, no ?

Reply:
It's used on component init, checking that previous selection.
By default they are not selected, false, so it can only turn On.
It can't be changed somewhere else while this is mounted.